src/core/reopt_inputs.jl

```diff
@@ -495,6 +506,22 @@ function setup_existing_chiller_inputs(s::AbstractScenario, max_sizes, min_sizes
     return nothing
 end
 
+
+function setup_absorption_chiller_inputs(s::AbstractScenario, max_sizes, min_sizes, cap_cost_slope, 
+    segmented_techs, n_segs_by_tech, seg_min_size, seg_max_size, seg_yint, cop, thermal_cop, om_cost_per_kw
+    )
+    max_sizes["AbsorptionChiller"] = s.absorption_chiller.max_kw
+    min_sizes["AbsorptionChiller"] = s.absorption_chiller.min_kw
+    update_cost_curve!(s.absorption_chiller, "AbsorptionChiller", s.financial,
```

Comment:
@zolanaj  does the AbsorptionChiller ever have a segmented cost curve? If not we can use the `effective_cost` function like [this](https://github.com/NREL/REopt.jl/blob/c9c0c474ab46bc38884aa970098eba4ff894fd49/src/core/reopt_inputs.jl#L502) to account for MACRS and remove the `AbsorptionChiller` fields that are always zero.

Reply:
> @zolanaj does the AbsorptionChiller ever have a segmented cost curve? If not we can use the `effective_cost` function like [this](https://github.com/NREL/REopt.jl/blob/c9c0c474ab46bc38884aa970098eba4ff894fd49/src/core/reopt_inputs.jl#L502) to account for MACRS and remove the `AbsorptionChiller` fields that are always zero.

@NLaws thanks for catching! no segmented cost curve exists to my knowledge.  I updated the cost curve in 63ad2df279e03a7bb210abd7265c428204df662b and will merge once I've confirmed that tests pass.

Reply:
Thanks @zolanaj ! Can you please also remove the unused fields from the `AbsorptionChiller` struct?

Reply:
Thanks for catching @NLaws!  Removed in 45f90074701069dc684d6bcf274c3d40a167ffcd.